### DISCUSSION TITLE
How to use link to the designated chromosome position?

### DISCUSSION THREAD

**Discussion by DerekZhou817** [GMOD/jbrowse-components#3353] [Q&A]
Hi,
I want to directly reach the specified interface by entering a link including chromosome location, because I need links each chromosome location(e.g. chr1  123456789) with my Jbowse.  The link of Browser in Jbrowse1 includes loc
http://www.XXXX.cn/jbrowse/?data=human&loc=chr2%3A110210626..110213920&tracks=DNA%2CncRNA. However，the link of Jbrowse2  is "http://XXX.cn/jbrowse2/?config=data%2Fconfig.json&session=share-Z4aVYzf-j8&password=eK81g".  How to use chromosome location(e.g. loc=chr2%3A110210626..110213920) in the link to display genome browser at designated location like jbrwose1. What can I do?
Thank you for your time and consideration.
Xu

**Comment by cmdcolin** [comment]
we have what we call the "URL query parameter API" that can do essentially the same thing from jbrowse 1 in jbrowse 2 https://jbrowse.org/jb2/docs/urlparams/

so for example this is a working link to a specific location in jbrowse 2

https://jbrowse.org/code/jb2/main/?config=test_data/volvox/config.json&loc=ctgA:1-800&assembly=volvox&tracks=gff3tabix_genes,volvox_filtered_vcf,volvox_microarray,volvox_cram

after visiting the link, your url will change into the "local session" style so it won't stay updated like jbrowse 1 (which is for reasons like that the url can get too big in jbrowse 2 if all the settings are stored accurately), but hopefully that helps!

**Reply by DerekZhou817** [reply]
Thank you for your reply. It's quite useful to me.
